examples/experimental/i3s-17-and-debug/app-debug.js

```diff
@@ -249,6 +251,15 @@ export default class App extends PureComponent {
     this.setState({debugOptions});
   }
 
+  handleValidateTile(tile) {
```

Comment:
Nit: `handleTileValidation`

Reply:
I think just `validateTile` will be better. You thoughts?